ecosystem/sep-0012.md

```diff
@@ -309,6 +343,55 @@ For example:
 }
 ```
 
+## Customer PUT Verification
+
+This endpoint allows servers to accept data values, usually confirmation codes, that verify a previously provided field via `PUT /customer`, such as `mobile_number` or `email_address`. Note that while fields such as `photo_proof_residence` or `notary_approval_of_photo_id` are verifications of other fields described in [SEP-9](#sep-0009.md), the server does not require the associated fields _before_ verification can be accomplished, so this endpoint would not be useful for such fields.
+
+Fields in the `VERIFICATION_REQUIRED` status require a request to this endpoint.
+```js
+{
+   "id": "d1ce2f48-3ff1-495d-9240-7a50d806cfed",
+   "status": "NEEDS_INFO",
+   "fields": {
+      "mobile_number": {
+         "description": "phone number of the customer",
+         "type": "string",
+         "status": "VERIFICATION_REQUIRED",
+      },
+   }
+}
+```
+
+### Request
+
+```
+PUT [KYC_SERVER || TRANSFER_SERVER]/customer/verification
+```
+
+Name | Type | Description
+-----|------|------------
+`id` | string | The ID of the customer as returned in the response of a previous `PUT` request.
+`*_verification` | string | One or more SEP-9 fields appended with `_verification`.
+
+```json
+{
+   "id": "391fb415-c223-4608-b2f5-dd1e91e3a986",
+   "mobile_number_verification": "2735021"
+}
+```
+
+### Response
+
+Success responses should return a `202 Accepted` status as well as a body matching the [GET /customer](#customer-get) response schema. The field statuses for which verifications were sent must be updated to either `PROCESSING` or `ACCEPTED`.
```

Comment:
💡 If the status is Accepted the I think 200 OK would be more appropriate because there is nothing else to do, processing has already completed. 202 is intended for when processing still needs to take place, which worse for the case where it is still processing, although you could even use 200 there too without much contention I think.

Reply:
Ah I didn't know that. I used 202 because thats what is return in `PUT /customer` responses. If `Accepted` should only be used when process is still necessary, do you think its worth adding 200 as an acceptable status for that endpoint as well, since 202 would only make sense if the customer was put into the `PROCESSING` status?

Reply:
I think 200 OK would suffice for both endpoints, because a full response is available in all cases. Clients should focus on using the customer status as the indicator not the http status code, because we can add new statuses. I get to make it always 200 would be a breaking change though, so another option is to say any 2xx response is fine.